tests/test_data/cookingwithjanica.com/cookingwithjanica_2.json

```diff
@@ -0,0 +1,75 @@
+{
+  "author": "Jessica Pinney",
+  "canonical_url": "https://cookingwithjanica.com/copycat-taco-bell-quesadilla-recipe/",
+  "site_name": "Cooking With Janica",
+  "host": "cookingwithjanica.com",
+  "language": "en-US",
+  "title": "Copycat Taco Bell Quesadilla Recipe",
+  "ingredients": [
+    "1/4 Cup Mayo",
+    "2 Tsp Minced Pickled Jalapenos",
+    "2 Tsp Pickled Jalapeno Juice",
+    "2/3 Tsp Sugar",
+    "1/2 Tsp Cumin",
+    "1/2 Tsp Paprika",
+    "1/8 Tsp Cayenne Pepper",
+    "1/8 Tsp Garlic Powder",
+    "1 Dash Salt",
+    "1 Chicken Breast",
+    "1 Cup Cheddar & Monterey Jack Cheese (If you can't find a bag that's a mix, just buy them separately and use 1/2 cup of each)",
+    "1 Slice American Cheese",
+    "2 Flour Tortillas (Burrito size)"
+  ],
+  "ingredient_groups": [
+    {
+      "ingredients": [
+        "1/4 Cup Mayo",
+        "2 Tsp Minced Pickled Jalapenos",
+        "2 Tsp Pickled Jalapeno Juice",
+        "2/3 Tsp Sugar",
+        "1/2 Tsp Cumin",
+        "1/2 Tsp Paprika",
+        "1/8 Tsp Cayenne Pepper",
+        "1/8 Tsp Garlic Powder",
+        "1 Dash Salt"
+      ],
+      "purpose": "Taco Bell Quesadilla Sauce Ingredients:"
```

Comment:
This presentation seems slightly different to #1386 (a PR that I've just approved).  I feel a bit pedantic about this, but I think this could reasonably be misunderstood as the actual recipe for a branded sauce.

I'm not certain whether that's strictly a problem for `recipe-scrapers` since it is the recipe provided on the website.  But I don't want to include it in our test data if it somehow indicates that this recipe website is doing anything wrong.

So I plan to contact the recipe site here to check.  It does seem that they work with some brands, and that they label sponsored/branded recipe posts -- of which this doesn't appear to be one -- and since they have a category of copycat recipes, I am nearly certain they know what they're doing and that this is all fine, but I think it's better to ask.

Reply:
Agreed, just swapped it out for a new test case. I'd be interested in knowing what the response of the site is if you receive one!

Reply:
No response so far, but I think it'd make sense to allow some more time for that; I'll send a subsequent enquiry in another few weeks or so, and/or when re-checking this PR.

I don't have any concerns with the technical/code content or updated test data of this PR - I'm a bit reluctant to approve it while still waiting to hear back though.